spec/syntax.md

```diff
@@ -285,13 +285,16 @@ Whitespace in _text_, including tabs, spaces, and newlines is significant and MU
 be preserved during formatting.
```

Comment:
We should add some text about the grammar to make clear our intentions

Reply:
By "the grammar", do you here mean the added `content-char`, or something else?

Reply:
That's what I mean: some text about the ABNF productions. Just a quick sentence or two. Probably not on this specific line, but somewhere

Reply:
Done.